iwyu_output.h

```diff
@@ -222,6 +224,45 @@ class OneIncludeOrForwardDeclareLine {
   const clang::NamedDecl* fwd_decl_ = nullptr;
 };
 
+// Namespace usage reporting is delayed until ResolvePendingAnalysis.
+// This structure stores the arguments to be passed to
+// RecordFullSymbolUse.
+struct NamespaceDeclUse {
```

Comment:
I think we can just use `OneUse` to buffer pending namespaces uses, instead of this custom type

Reply:
You're right - It does hold much of the same data, but has quite a bit more too. I could see this being confusing. Maybe a tuple might be prefered here?